eng/pipelines/templates/jobs/new.smoke.tests.yml

```diff
@@ -0,0 +1,42 @@
+parameters:
+  - name: Daily
+    type: boolean
+    default: true
+  - name: Artifact
+    type: object
+    default: {}
+  - name: ArtifactName
+    type: string
+    default: ""
+  - name: TimeoutInMinutes
+    type: number
+    default: 60
+
+jobs:
+  - job: SmokeTest
+    steps:
+      - ${{ if eq(parameters.Daily, true) }}:
```

Comment:
Why write these to disk as opposed to just read the values directly?

Reply:
I think as a boundary between steps/processes it makes local development a lot easier to pass things around via file. You can inspect the output more easily when writing your own tools, and also modify it to test various cases, without needing to re-run the package properties script and/or worry about variables persisting across shell sessions.

Reply:
We are already running PS code why not just call the Get-AllPkgProperties directly and loop over those instead of going through the process of writing them to disk and then reading them from disk again.

Reply:
The step that belongs to the language teams won't necessarily be powershell. I think the pipeline config would be simpler to have a common set of steps and then a simple drop-in yaml step for the language smoke test binary, without requiring extra script handling for passing data to the command line? Otherwise it's going to be harder to move this yaml into eng/common.